### DISCUSSION TITLE
offer WisdPi 5Gbps ethernet adapter WP-UT5

### DISCUSSION THREAD

**Discussion by wisdpi** [bb-qq/r8152#430] [General]
@bb-qq  we don't know how to reach you. We hope you can contact us via email (admin@wisdpi.com). 
We would like to offer WisdPi 5Gbps ethernet adapter WP-UT5 for testing its compatibility with 5Gbps on the Synology system.

**Comment by bb-qq** [comment]
@wisdpi, thank you for your offer to provide the product.
I have created an Amazon wish list and would be happy to have you send me items through it.

https://www.amazon.com/hz/wishlist/ls/26G32SHBO122O

**Reply by wisdpi** [reply]
Can you email me the address directly? We would like to send it to you directly. email: admin@wisdpi.com

**Comment by burtgolf-ux** [comment]
@bb-qq @wisdpi Did this adapter ever get confirmed as working? Thanks!

**Reply by bb-qq** [reply]
Ultimately, since I haven't received this device myself, I haven't been able to verify its operation. Please refer to community reports.
https://github.com/bb-qq/r8152/issues?q=is%3Aissue%20WP-UT5
